### ISSUE TITLE
Page properties of V3 page not opening because of NPE

### ISSUE THREAD

**Issue by tenitram** [adobe/aem-core-wcm-components#1690]
## Bug Report

**Current Behavior**
Page properties dialog stopped working for Page v3. When opening page properties of /content/core-components-examples/library/page-authoring/image from core-components-examples

**Expected behavior/code**
Page properties open without error.

**Environment**
- AEM version and patch level AEM 6.5.9.0
- Core Components version 2.17.2

**Possible Solution**
It seems that new featured image part is causing the problem. I installed fresh AEM 6.5, then latest SP, then core components and the field file of the type: cq/gui/components/authoring/dialog/fileupload from Images tab is throwing Null Pointer Exception when opening page properties of existing page. When creating new page, it renders correctly. It throws exception on the following code: 
`String compositePath = action.substring(contextResourcePath.length());` 
(line 157 of /libs/cq/gui/components/authoring/dialog/fileupload/render.jsp) which assumes that url has got suffix. It is true for creating new page, but when opening existing one, path is passed as query parameter: http://localhost:6302/mnt/overlay/wcm/core/content/sites/properties.html?item=%2Fcontent%2Fcore-components-examples%2Flibrary%2Fpage-authoring%2Fteaser

**Additional context / Screenshots**
Is it only me having this issue? Maybe other field type (granite/ui/components/coral/foundation/form/fileupload - the one used for thumbnail image on the same tab) could be used?
Is it know issue of AEM?

**Stacktrace**
java.lang.NullPointerException
	at org.apache.jsp.libs.cq.gui.components.authoring.dialog.fileupload.render_jsp._jspService(render_jsp.java:259)
	at org.apache.sling.scripting.jsp.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.sling.scripting.jsp.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:502)
	at org.apache.sling.scripting.jsp.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:449)
	at org.apache.sling.scripting.jsp.JspScriptEngineFactory.callJsp(JspScriptEngineFactory.java:339)
	at org.apache.sling.scripting.jsp.JspScriptEngineFactory.access$100(JspScriptEngineFactory.java:97)
	at org.apache.sling.scripting.jsp.JspScriptEngineFactory$JspScriptEngine.eval(JspScriptEngineFactory.java:600)
	at org.apache.sling.scripting.core.impl.DefaultSlingScript.call(DefaultSlingScript.java:388)
	at org.apache.sling.scripting.core.impl.DefaultSlingScript.eval(DefaultSlingScript.java:184)
	at org.apache.sling.scripting.core.impl.DefaultSlingScript.service(DefaultSlingScript.java:491)
	at com.adobe.granite.ui.components.impl.BaseComponentHelper.call(BaseComponentHelper.java:568)
	at com.adobe.granite.ui.components.ComponentHelper.call(ComponentHelper.java:758)
	at org.apache.jsp.libs.granite.ui.components.coral.foundation.form.field.field_jsp._jspService(field_jsp.java:304)
	at org.apache.sling.scripting.jsp.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)

....
```
739186 TIMER_END{10946,/libs/granite/ui/components/coral/foundation/form/field/init.jsp}
 743907 LOG SCRIPT ERROR: An exception occurred processing JSP page /libs/cq/gui/components/authoring/dialog/fileupload/render.jsp at line 157
 744867 LOG SCRIPT ERROR: An exception occurred processing JSP page /libs/granite/ui/components/coral/foundation/form/field/field.jsp at line 221
 744876 TIMER_END{23621,/libs/granite/ui/components/coral/foundation/form/field/field.jsp#92}
```

**Comment by jckautzmann** [comment]
@adobe export issue to Jira project SITES as Story

**Comment by bpauli** [comment]
This issue will be fixed with the release of the SP 6.5.10 which is scheduled for the 26th of August. Meanwhile AEM 6.5 customers have to keep using the page V2 component version.
